index.html

```diff
@@ -1,85 +1,79 @@
+---
+# MagisterJS Project Website
```

Comment:
Just to let you know @lieuwex, this is called [front matter](https://jekyllrb.com/docs/frontmatter) and makes Jekyll parse the `index.html` file.